### DISCUSSION TITLE
Installation guide

### DISCUSSION THREAD

**Discussion by unfairDude** [zan8in/afrog#56] [Q&A]
How to install?

**Comment by zan8in** [comment]
**golang compile**
1. go >= 1.19
2. git clone https://github.com/zan8in/afrog
3. cd afrog
4. go mod tidy
5. go run cmd/afrog/main.go

**download release**

[↓Download](https://github.com/zan8in/afrog/releases)

![image](https://user-images.githubusercontent.com/4088460/234742731-05920625-e6c6-457a-8a3e-5f5edd194504.png)
